src/PrestaShopBundle/Resources/views/Admin/Module/manage.html.twig

```diff
@@ -1,107 +1,77 @@
-                  data-content="{{ "These are all the modules you’ve added to your shop, either by buying it from PrestaShop Addons, or by uploading it directly."|trans({}, 'Admin.Modules.Help') }}">
-                </span>
-            </div>
-            {% include 'PrestaShopBundle:Admin/Module/Includes:grid_manage_installed.html.twig' with { 'modules': modules.modules, 'display_type': 'list', 'origin': 'manage', 'id': 'all' } %}
+  <div class="row justify-content-center">
+    <div class="col-lg-10">
+      {# Bulk Action confirm modal #}
+      {% include 'PrestaShopBundle:Admin/Module/Includes:modal_confirm_bulk_action.html.twig' %}
+      {# PrestaTrust modal #}
+      {% include 'PrestaShopBundle:Admin/Module/Includes:modal_confirm_prestatrust.html.twig' %}
+      {# Addons connect modal with level authorization#}
+      {% include 'PrestaShopBundle:Admin/Module/Includes:modal_addons_connect.html.twig' with { 'level' : level, 'errorMessage' : errorMessage } %}
+      {# Contains toolbar-nav for module page with level authorization #}
+      {% include 'PrestaShopBundle:Admin/Module/Includes:modal_import.html.twig' with { 'level' : level, 'errorMessage' : errorMessage } %}
+      {# Contains menu with Selection/Categories/Popular and Tag Search #}
+      {% include 'PrestaShopBundle:Admin/Module/Includes:menu_top.html.twig' with { 'topMenuData': topMenuData, 'bulkActions': bulkActions } %}
 
-            <hr class="top-menu-separator">
-
-            <div class="module-short-list">
-                <span id="built-in_modules" class="module-search-result-wording">{{ '%nbModules% built-in modules'|trans({'%nbModules%' : modules.native_modules|length}, 'Admin.Modules.Feature') }}</span>
-                <span class="help-box" data-toggle="popover"
-                  data-title="{{ "Built-in modules"|trans({}, 'Admin.Modules.Feature') }}"
-                  data-content="{{ "These modules from PrestaShop are preinstalled when you set-up your shop. They cover the basics of e-commerce and comes for free."|trans({}, 'Admin.Modules.Help') }}">
-                </span>
-            </div>
-            {% include 'PrestaShopBundle:Admin/Module/Includes:grid_manage_installed.html.twig' with { 'modules': modules.native_modules, 'display_type': 'list', 'origin': 'manage', 'id': 'native' } %}
+      <div id="module-recently-used-list" class="module-short-list">
+        <span id="recently-used_modules" class="module-search-result-wording">{{ 'Recently Used' | trans({}, 'Admin.Modules.Feature') }}</span>
+        <div class="modules-list" data-name="recently-used"></div>
+      </div>
 
-            <hr class="top-menu-separator">
+      {% for category in categories.subMenu %}
+        <div class="module-short-list">
+          <span id="{{ category.tab }}_modules" class="module-search-result-wording">{{ category.name | trans({}, 'Admin.Modules.Feature') }}</span>
```

Comment:
What is the wording here, category.name?

Reply:
Categories came from Addons API.
`Administration` for example.